assets/sass/_theme/sections/locations.sass

```
@@ -1,51 +1,76 @@
 .locations
-    &--list
-        .location + .location
+    .location
+        display: flex
+        flex-direction: column-reverse
+        gap: calc(var(--grid-gutter) / 2) var(--grid-gutter)
+        justify-content: flex-end
+        position: relative
+        + .location
             margin-top: $spacing-4
+        a
+            @include stretched-link(after)
+            text-decoration: none
+        .more
+            @include hover-translate-icon(after)
+            @include meta
+            @include icon(arrow-right-line, after)
+                margin-left: $spacing-2
+            margin-top: $spacing-1
+        .media
+            width: columns(6)
+            picture
+                aspect-ratio: 3/2
+                display: block
+            img
+                display: block
+                height: 100%
+                object-fit: cover
+        &-description
+            margin-top: $spacing-1
+
+    &--list
         @include media-breakpoint-up(desktop)
             .location
+                border-bottom: 1px solid var(--color-border)
                 flex-direction: row
```

```suggestion
                flex-direction: row-reverse
```
comme ça tu peux enlever tout ça : 
```
                &-content
                    order: 2
                .media
                    order: 1
```